### DISCUSSION TITLE
🐛 [BUG]水波图配置悬浮提示没有生效

### DISCUSSION THREAD

**Discussion by is-cool** [antvis/G2Plot#3555] [Q&A] [answered]
### 🐛 [Bug description [Please make everyone to understand it]
](https://codesandbox.io/s/fervent-euler-5eo8oh)

### 🚑 Any additional [like screenshots]

* **G2Plot Version**: 2.4.29
* **Platform**:

**Comment by hustcc** [comment, marked as answer]
水波图当前是没有设置 tooltip 的，因为信息量很少，如果需要显示其他额外信息，建议可以外置监听事件来实现。
